examples/adzerk.amp.html

```diff
@@ -14,21 +14,21 @@
 
   <amp-ad width=300 height=250
       type="adzerk"
-      src="https://adzerk.com?id=0">
+      src="https://engine.adzerk.net/amp?id=0">
```

Comment:
you might not want the URL to be from the pub, we can hard code in the extension.
also, the `id` param is only for dev purpose.